ubuntu/jammy/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG ARCH=
+FROM ${ARCH}ubuntu:impish
```

Comment:
??

Reply:
It was a draft PR for @ylobankov to show the new layer structure. Fixed.